lib/gui/app/models/store.ts

```diff
@@ -200,7 +200,7 @@ function storeReducer(
 							constraints.isDriveValid(drive, image) &&
 							!drive.isReadOnly &&
 							constraints.isDriveSizeRecommended(drive, image) &&
-							// We don't want to auto-select large drives execpt is autoSelectAllDrives is true
+							// We don't want to auto-select large drives except is autoSelectAllDrives is true
```

Comment:
"except is" should probably be "except if" ?

Reply:
I'll fix this this morning